starter-code/src/functions-and-arrays.js

```diff
@@ -1,15 +1,49 @@
 // Iteration #1: Find the maximum
+console.log(Math.max(3, 2));
+//or
+function maxTowNumbers(n1, n2){
+  if(n1 > n2){
+    return n1;
+  }else{
+    return n2;
+  }
+}
+
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+functions findLongestWord(words){
```

Comment:
*function* not functions
I assume it was a typo, but pay attention to this because it was breakig your whole code